### ISSUE TITLE
Settings menu disappears on click when video is playing

### ISSUE THREAD

**Issue by papigers** [sampotts/plyr#1122]
### Expected behaviour
When clicking on the settings to change quality, or play speed, the menu disappears, clicking on the menu again, shows the specific settings menu, as I clicked before.
The issue doesn't reproduce when video is paused, clicking on the settings button and then on Quality, will show me the quality options without the menu disappearing.

### Actual behaviour
The menu shouldn't disappear, even when video is playing.

### Environment
Tested on the Plyr website: https://plyr.io/

- Browser: Chrome
- Version: Version 67.0.3396.99 (Official Build) (64-bit)
- Operating System: Windows 10
- Version: 1803

### Steps to reproduce
- When video is playing, click on the settings button.
- Next, click on one of the options, say Quality.
- The menu should disappear now, clicking on the settings button again, should display the quality options.
- Doing the same process when the video is paused, the menu would not disappear on click.

**Comment by friday** [comment]
What's the output of `player.touch`? This happens when I use the "device" emulator in Chrome, but otherwise not.

**Comment by papigers** [comment]
`player.touch` is `true`. My computer has a touchscreen so that makes sense. So you say the bug doesn't reproduce otherwise?

**Comment by friday** [comment]
Yes. It's most likely the same issue as #999 (but this one is described better so if anything we should close that one).

Neither Sam or me have much time right now, so help would be welcome.

**Comment by friday** [comment]
Also related: #1067

**Comment by jamesoflol** [comment]
I've been investigating this issue. It is present on phone and on Chrome dev tools mobile emulator.

There's also a directly related issue: Tapping on the video to show controls intermittently does nothing. Always works on the second tap

A simple fix for both of these is to remove the event handler for `focusin` and `focusout` at listeners.js:705.

The blocks at both listeners.js:272 (on container mousemove, touchmove etc) and listeners.js:705 (on focusin focusout) seem to implement the same functionality of trying to show/hide controls as needed. But line 705 has the added functionality of adding a `noTransition` class. I can't figure out why this noTransition class is required. After removing the entire block at 705, the issues described here are fixed, and I also seem to have no issues with scrolling of parent element.

**Comment by sampotts** [comment]
The `noTransition` class does exactly that, it means the controls are shown _instantly_ rather than slowly transitioning in. I'd have to have a look. The controls logic was re-written a few times and as there's so many scenarios it's always a tricky one.
